### PR TITLE
Create TestCafe aXe tests

### PR DESCRIPTION
**Pull request by Raushen**
Adding packages and the first aXe regression test.

### CODE REVIEW

File: playground/jquery.html [hunk 1]
```diff
@@ -30,16 +30,21 @@
...
 </head>
 <body>
-    <div id="button"></div>
-    <script>
-        $(function() {
-            $("#button").dxButton({
-                text: 'Click me!',
-                onClick: function() { alert("clicked"); }
+    <div role="main">
+        <h1 style="position: fixed; left: 0; top: 0; clip: rect(1px, 1px, 1px, 1px);">Test header</h1>
```
Comment (pomahtri) [inline playground/jquery.html:37]: why do we need test header?

File: testing/testcafe/tests/container.html [hunk 2]
```diff
@@ -12,12 +12,16 @@
     <link rel="dx-theme" data-theme="material.blue.dark" href="../../../artifacts/css/dx.material.blue.dark.css" data-active="false">
 
     <script type="text/javascript" src="../../../artifacts/js/jquery.min.js"></script>
-    <script type="text/javascript" src="../../../artifacts/js/dx.all.js"></script>
+    <script type="text/javascript" src="../../../node_modules/axe-core/axe.min.js"></script>
```
Comment (pomahtri) [inline testing/testcafe/tests/container.html:16]: let's add devextreme scripts back :)

File: testing/testcafe/tests/container.html [hunk 3]
```diff
@@ -12,12 +12,16 @@
...
 </head>
 <body>
-    <div id="container">
-    </div>
-    <div id="otherContainer">
+    <div role="main">
+        <h1 style="position: fixed; left: 0; top: 0; clip: rect(1px, 1px, 1px, 1px);">Test header</h1>
```
Comment (pomahtri) [inline testing/testcafe/tests/container.html:20]: Why do we need test header?

File: testing/testcafe/tests/dataGrid/grouping/grouping.ts [hunk 4]
```diff
@@ -1,9 +1,12 @@
 import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+/* eslint-disable spellcheck/spell-checker */
```
Comment (pomahtri) [inline testing/testcafe/tests/dataGrid/grouping/grouping.ts]: Looks like we'll write it very often, let's add `axe` word to spell-checker dictionary in eslintrc.json, and move it to `eslint-config-devextreme` in future

File: testing/testcafe/tests/dataGrid/grouping/grouping.ts [hunk 5]
```diff
@@ -259,3 +262,38 @@ safeSizeTest('Empty header message should appear when all columns grouped and se
...
+    rules: {
+      'color-contrast': { enabled: false },
+    },
+  });
+
+  await t
+    .expect(error)
+    .eql(null)
+    .expect(results.violations.length === 0)
+    .ok(createReport(results.violations));
```
Comment (pomahtri) [inline testing/testcafe/tests/dataGrid/grouping/grouping.ts]: Too many `expect`s for one check (that axe doesn't see any error), maybe move it to helper?

File: playground/jquery.html (hunk 1 above)
Comment (Raushen) [reply playground/jquery.html:37]: It is an accessibility requirement and aXe tracks this point.

File: testing/testcafe/tests/container.html (hunk 2 above)
Comment (Raushen) [reply testing/testcafe/tests/container.html:16]: Of cause, I already returned)

File: testing/testcafe/tests/container.html (hunk 3 above)
Comment (Raushen) [reply testing/testcafe/tests/container.html:20]: It is an accessibility requirement and aXe tracks this point.

**Review by pomahtri — APPROVED** [review]

**Review by andrewmakarov — APPROVED** [review]

**Pull request merged**